devtools/conda-envs/CI_env.yaml

```diff
@@ -6,7 +6,6 @@ dependencies:
   - numpy
   - matplotlib
   - rdkit
-  - boost-cpp
```

Comment:
not sure why this was there, removed it & CI still pases

Reply:
Probably historical issues with boost via rdkit